react/features/analytics/functions.ts

```diff
@@ -232,7 +233,14 @@ export function initAnalytics(store: IStore, handlers: Array<Object>) {
         }
     }
 
-    analytics.addPermanentProperties(permanentProperties);
+    analytics.addPermanentProperties({
+        ...permanentProperties,
+        ...getState()['features/analytics'].initialPermanentProperties
+    });
+
+    // Clear the initialPermanentProperties since we don't need them anymore
+    dispatch({ type: CLEAR_INITIAL_PERMANENT_PROPERTIES });
```

Comment:
Does it hurt to have them there until the initialized flag goes back to false, since we only used them in a single place?

Reply:
It shouldn't hurt. I've changed it!